sys/arm64/arm64/swtch.S

```diff
@@ -213,6 +213,22 @@ ENTRY(fork_trampoline)
 	mov	fp, #0	/* Stack traceback stops here. */
 	bl	_C_LABEL(fork_exit)
 
+#if __has_feature(capabilities)
+	ldp	c2, c3, [sp, #TF_X + 2 * 16]
+	ldp	c4, c5, [sp, #TF_X + 4 * 16]
+	ldp	c6, c7, [sp, #TF_X + 6 * 16]
+	ldp	c8, c9, [sp, #TF_X + 8 * 16]
+	ldp	c10, c11, [sp, #TF_X + 10 * 16]
+	ldp	c12, c13, [sp, #TF_X + 12 * 16]
+	ldp	c14, c15, [sp, #TF_X + 14 * 16]
+	ldp	c16, c17, [sp, #TF_X + 16 * 16]
+	ldr	     c19, [sp, #TF_X + 19 * 16]
+	ldp	c20, c21, [sp, #TF_X + 20 * 16]
+	ldp	c22, c23, [sp, #TF_X + 22 * 16]
+	ldp	c24, c25, [sp, #TF_X + 24 * 16]
+	ldp	c26, c27, [sp, #TF_X + 26 * 16]
+	ldp	c28, c29, [sp, #TF_X + 28 * 16]
+#else
 	/* Restore the registers other than x0 and x1 */
```

Comment:
```suggestion
	/* Restore the registers other than x0 and x1 */
#if __has_feature(capabilities)
	ldp	c2, c3, [sp, #TF_X + 2 * 16]
	ldp	c4, c5, [sp, #TF_X + 4 * 16]
	ldp	c6, c7, [sp, #TF_X + 6 * 16]
	ldp	c8, c9, [sp, #TF_X + 8 * 16]
	ldp	c10, c11, [sp, #TF_X + 10 * 16]
	ldp	c12, c13, [sp, #TF_X + 12 * 16]
	ldp	c14, c15, [sp, #TF_X + 14 * 16]
	ldp	c16, c17, [sp, #TF_X + 16 * 16]
	ldr	     c19, [sp, #TF_X + 19 * 16]
	ldp	c20, c21, [sp, #TF_X + 20 * 16]
	ldp	c22, c23, [sp, #TF_X + 22 * 16]
	ldp	c24, c25, [sp, #TF_X + 24 * 16]
	ldp	c26, c27, [sp, #TF_X + 26 * 16]
	ldp	c28, c29, [sp, #TF_X + 28 * 16]
#else
```

to match the other places, or systematically duplicate the comments on each side of the #if and change x to c.

Reply:
I don't know why GitHub is rendering the diff in such a strange way, it just moves the line from the bottom to the top...